lib/servo.js

```diff
@@ -275,8 +274,8 @@ Servo.prototype.to = function(degrees, time, rate) {
   if (this.invert) {
     degrees = Board.map(
       degrees,
-      this.range[0], this.range[1],
-      this.range[1], this.range[0]
+      0, 180,
+      180, 0
```

Comment:
I bet it is. On my hexapod I had started specifying absurdly loose range values just to get the servo to move when using a global coordinate space for the legs. I thought the problem was with my IK formulas, but I traced it back to here. It wasn't a problem when I was using a joint coordinate space for specifying positions because the offsets were small and I wasn't using ranges yet. It's a huge relief to have figured this one out.